src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -593,14 +585,16 @@ private function sortGroupElements(array $a, array $b): int
     {
         $selectedSortAlgorithm = $this->configuration['sort_algorithm'];
 
-        if (
-            self::SORT_ALPHA === $selectedSortAlgorithm || self::SORT_ALPHA_CASE_INSENSITIVE === $selectedSortAlgorithm
-        ) {
-            return strcasecmp($a['name'], $b['name']);
-        }
+        if (self::SORT_ALPHA === $selectedSortAlgorithm) {
+            $isCaseSensitive = $this->configuration['case_sensitive'];
 
-        if (self::SORT_ALPHA_CASE_SENSITIVE === $selectedSortAlgorithm) {
-            return strcmp($a['name'], $b['name']);
+            if (false === $isCaseSensitive) {
+                return strcasecmp($a['name'], $b['name']);
+            }
+
+            if ($isCaseSensitive) {
+                return strcmp($a['name'], $b['name']);
+            }
```

Comment:
We have only bi-way path here, so ternary is good enough (whether should be multiline is a cosmetic detail):
```suggestion
            return $isCaseSensitive ? strcmp($a['name'], $b['name']) : strcasecmp($a['name'], $b['name']);
```

Reply:
Done